modules/transactions.js

```diff
@@ -5,12 +5,15 @@ var async = require('async');
 var constants = require('../helpers/constants.js');
 var crypto = require('crypto');
 var extend = require('extend');
+var apiCodes = require('../helpers/apiCodes.js');
+var ApiError = require('../helpers/apiError.js');
```

Comment:
casing

Reply:
apiCodes is an object, ApiError is a constructor function. I think this is the right casing according to javascript best practices.